src/scenes/home/contact/contact.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+import styles from './contact.css';
+
+
+class Contact extends Component {
+  render() {
+    return (
+      <div className={styles.wrapper}>
```

Comment:
This isn't a review requested, but maybe an idea to make this code (and the stylesheets) a bit smaller - could you potentially just use the <Section> component with the following passed props:
```
<Section
  title="Contact Us"
  className="wrapper"
  headingLines="true"
  headingTheme="dark"
/>
```